packages/TelegramClient-Tests.package/TCTMocks.class/class/mockPrivateChat.st

```diff
@@ -0,0 +1,13 @@
+chat
+mockPrivateChat
+
+	|chat|
```

Comment:
In most places we have spaces around the temporary variables. I suggest just sticking with it.